### DISCUSSION TITLE
The Default/Preferences file is growing

### DISCUSSION THREAD

**Discussion by dim22-star** [ultrafunkamsterdam/undetected-chromedriver#1343] [General]
The Default/Preferences file increases to several GB. I know that this file contains browser settings, I periodically have to delete it.
Maybe the problem is in the code __init__.py starting from line 402 ? Or am I wrong?
`        # fix exit_type flag to prevent tab-restore nag
        try:
            with open(
                os.path.join(user_data_dir, "Default/Preferences"),
                encoding="latin1",
                mode="r+",
            ) as fs:
                config = json.load(fs)
                if config["profile"]["exit_type"] is not None:
                    # fixing the restore-tabs-nag
                    config["profile"]["exit_type"] = None
                fs.seek(0, 0)
                json.dump(config, fs)
                fs.truncate()  # the file might be shorter
                logger.debug("fixed exit_type flag")
        except Exception as e:
            logger.debug("did not find a bad exit_type flag ")`

**Comment by ultrafunkamsterdam** [comment]
I noticed this issue myself also last week. It was Soo large it didn't even load any longer. Don't have a fix. For me Deleting the file and restarting chrome creates a fresh one . Settings included
